WordPress/twentytwentyfive
# Big heading and search form pattern

**Description**
This PR adds a new pattern: Search - Big heading and search form. Also, additional style settings were added to the core/search block on the theme.json.

Additional notes:
[Design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1781&node-type=SYMBOL&t=9PtDqHUkO22FE08W-0)
[Issue](https://github.com/WordPress/twentytwentyfive/issues/71). 

**Screenshots**

<img width="1651" alt="Screenshot 2024-08-30 at 12 42 16 PM" src="https://github.com/user-attachments/assets/1ce31a49-4c3c-4f1f-a71b-1cd2636365be">


**Testing Instructions**

1. Create a page.
2. Add the Search - big  heading and search form pattern
3. Confirm everything is looking like [the designs](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1781&node-type=SYMBOL&t=9PtDqHUkO22FE08W-0).
